### ISSUE TITLE
Missing 'promise' dependency

### ISSUE THREAD

**Issue by bswatson** [lkwdwrd/generator-vvv#49]
This is against the version1 branch -

```
yo vvv:create
module.js:341
    throw err;
    ^

Error: Cannot find module 'promise'
    at Function.Module._resolveFilename (module.js:339:15)
    at Function.Module._load (module.js:290:25)
    at Module.require (module.js:367:17)
    at require (internal/module.js:16:19)
    at Object.<anonymous> (/Users/watson/Development/tools/generator-vvv/lib/util/rc-config.js:4:16)
    at Module._compile (module.js:413:34)
    at Object.Module._extensions..js (module.js:422:10)
    at Module.load (module.js:357:32)
    at Function.Module._load (module.js:314:12)
    at Module.require (module.js:367:17)
```

**Comment by lkwdwrd** [comment]
Hmm, I'll take a look. I switched promise libraries from 'promise' to 'bluebird' so that's likely what's going on. Will ping back when I take a look here.

**Comment by lkwdwrd** [comment]
Resolved by #50
